faasmcli/faasmcli/tasks/release.py

```diff
@@ -163,43 +167,6 @@ def create(ctx):
     )
 
 
-@task
-def upload(ctx, which=None):
```

Comment:
We no longer bundle artifacts with the Faasm release, but instead with the relevant container build for the other repos